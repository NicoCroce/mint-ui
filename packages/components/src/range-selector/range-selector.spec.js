import * as React from "react";
import { render, cleanup } from "@testing-library/react";
import RangeSelector from "./";

export const TestRangeSelector = () => {
	
	return (
		<RangeSelector />
	);
};

afterEach(cleanup);

describe("RangeSelector tests", () => {
	test("renders correcty", () => {
		const { asFragment } = render(<TestRangeSelector />);
		expect(asFragment()).toMatchSnapshot();
	});
	
});
