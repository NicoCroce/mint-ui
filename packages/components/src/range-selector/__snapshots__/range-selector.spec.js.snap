// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeSelector tests renders correcty 1`] = `
<DocumentFragment>
  <div
    class="range-selector__RangeSelectorWrapper-sc-1s3z4an-0 hMwAGm"
  >
    <div
      class="range-selector__TrackWrapper-sc-1s3z4an-4 hRbFEK"
      style="transform: scale(1); cursor: pointer;"
    >
      <div
        class="range-selector__TrackLine-sc-1s3z4an-3 bKBFxO"
        max="100"
        min="0"
      >
        <div
          class="range-selector__StyledThumb-sc-1s3z4an-5 ilgMZk"
          disabled=""
          style="transform: translate(NaNpx, NaNpx);"
        >
          <div
            class="range-selector__StyledThumbLable-sc-1s3z4an-6 hueBnI"
          >
            0
          </div>
        </div>
      </div>
    </div>
    <div />
  </div>
</DocumentFragment>
`;
